partiql-ast/src/test/kotlin/org/partiql/ast/sql/SqlDialectTest.kt

```diff
@@ -430,1776 +481,2177 @@ class SqlDialectTest {
-                }
-            },
-            expect("(1, 2, 3)") {
-                exprCollection {
-                    type = Expr.Collection.Type.LIST
-                    values += exprLit(int32Value(1))
-                    values += exprLit(int32Value(2))
-                    values += exprLit(int32Value(3))
-                }
-            },
-            expect("SEXP ()") {
-                exprCollection {
-                    type = Expr.Collection.Type.SEXP
-                }
-            },
-            expect("SEXP (1, 2, 3)") {
-                exprCollection {
-                    type = Expr.Collection.Type.SEXP
-                    values += exprLit(int32Value(1))
-                    values += exprLit(int32Value(2))
-                    values += exprLit(int32Value(3))
-                }
-            },
+            expect(
+                "<<>>",
+                exprBag(values = emptyList())
+            ),
+            expect(
+                "<<1, 2, 3>>",
+                exprBag(
+                    values = listOf(
+                        exprLit(int32Value(1)),
+                        exprLit(int32Value(2)),
+                        exprLit(int32Value(3))
+                    )
+                )
+            ),
+            expect(
+                "[]",
+                exprArray(values = emptyList())
+            ),
+            expect(
+                "[1, 2, 3]",
+                exprArray(
+                    values = listOf(
+                        exprLit(int32Value(1)),
+                        exprLit(int32Value(2)),
+                        exprLit(int32Value(3))
+                    )
+                )
+            ),
+            expect(
+                "VALUES ()",
+                exprValues(
+                    rows = listOf(
+                        exprRowValue(
+                            values = emptyList()
+                        )
+                    )
+                )
+            ),
+            expect(
+                "VALUES (1, 2, 3)",
+                exprValues(
+                    rows = listOf(
+                        exprRowValue(
+                            values = listOf(
+                                exprLit(int32Value(1)),
+                                exprLit(int32Value(2)),
+                                exprLit(int32Value(3))
+                            )
+                        )
+                    )
+                )
+            ),
+            expect(
+                "()",
+                exprRowValue(
+                    values = emptyList()
+                )
+            ),
+            expect(
+                "(1, 2, 3)",
+                exprRowValue(
+                    values = listOf(
+                        exprLit(int32Value(1)),
+                        exprLit(int32Value(2)),
+                        exprLit(int32Value(3))
+                    )
+                )
+            ),
+            // SEXP not in v1
```

Comment:
(self-review) the SEXP constructor was removed in the v1 AST. IIRC, these were the only tests I commented out from `SqlDialect`.